chain/network separation and nft/credit separtion

kriii opened the pull request:
- Separate chain into their networks. (i.e., change `ETH`, `ETH-TEST` to `ETH/homestead`, `ETH/goerli` resp.)
- Separate `asset` into `nft` and `credit`.

platfowner commented on src/constants.ts:
```diff
@@ -10,3 +10,17 @@ export const AIN_BLOCKCHAIN_ENDPOINT = NODE_ENV === 'prod'
+export const SUPPORTED_NETWORKS: { [chain: string]: Array<string> } = {
+  ETH: [ETH_NETWORKS.HOMESTEAD, ETH_NETWORKS.GOERLI],
+};
+export const DEFAULT_NETWORK: { [chain: string]: string } = {
```
question: do we need a default value for 'ETH-TEST' as well here?
'ETH-TEST': ETH_NETWORKS.GOERLI

platfowner replied:
Ah, ETH-TEST is not supported anymore by the upper versions of this package. Got it, thanks! 👌

platfowner commented on src/credit.ts:
```diff
@@ -0,0 +1,210 @@
+
+// TODO(kriii): Objectify params?
+export default class Credit extends AinftBase {
+  createAppCredit(
```
question: should the corresponding original code be removed from asset.ts?

kriii replied:
No, I think original codes should remain for backward compatibility.

platfowner replied:
As old users can use old versions of the npm package, I think we can just remove old libraries. What do you guys think? (The ainft-server rest APIs, however need to be kept for old npm packages.) @kriii @woomurf @ehgmsdk20

platfowner replied:
If we want to guide the users of the package to migrate to the new structure, how about doing it with the next release? For users's reference, I think it's fine to just comment out the old code. But do we need to allow them to keep using the old APIs?

ehgmsdk20 replied:
I think there is no need to leave the original code, because users are using the old version of the library and there is no problem in continuing to operate as long as the old rest apis used there remains. Rather, it seems that we have to remove the existing code so that when users upgrade the package, an error saying that the function does not exist appears and we can force migration.

platfowner commented on src/nft.ts:
```diff
@@ -0,0 +1,110 @@
+} from './types';
+
+export default class Nft extends AinftBase {
+  addNftSymbol({
```
ditto?

kriii commented:
To rebase upstream changes, I reworked this and force-pushed. PTAL.

- As priorly discussed in the meeting, I removed `asset.ts`.
- Adjusted API routing paths to correspond with nft-server.
- I found some style changes, so I reverted it. Is that the intended one?

akaster99 commented:
> To rebase upstream changes, I reworked this and force-pushed. PTAL.
> 
> * As priorly discussed in the meeting, removed `asset.ts`.
> * Adjusted API routing paths to correspond with nft-server.
> * Some style changes were found, so reverted it. Is that change are intended?

style changes occurred by format setting on save at vscode. it was not intended.

platfowner reviewed (approved):
LGTM
Thanks!

kriii commented:
Now merged! Thanks for the reviews!
